Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -0,0 +1,138 @@
+  cost: 0
+  exclusive: true
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutHeadSecurity
+    - !type:CharacterDepartmentRequirement
+      departments:
+        - Security
+  items:
+    - LPPClothingHeadHatRedBeanie
+
+#Шея
```

yamllint

[warning] missing starting space in comment

(comments)